lib/imgapi.js

```diff
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright 2019 Joyent, Inc.
  */
 
 /*
```

Comment:
There was this hunk from commit 3d668a961ea2fff59afc41d78a555e02cf0e0c24

```
--- a/lib/imgapi.js
+++ b/lib/imgapi.js
@@ -9,7 +9,7 @@
  */

 /*
- * Client library for the SDC Image API (IMGAPI).
+ * Client library for the TritonDC Image API (IMGAPI).
  *
  * Usage without auth (e.g. when talking to in-SDC IMGAPI on admin network):
  *
```

Obviously minor, but any reason to not include it?

Reply:
Nope, no reason - we'll take that in.